doc/reference/storage_powerflex.md

```diff
@@ -0,0 +1,151 @@
+(storage-powerflex)=
+# Dell PowerFlex - `powerflex`
+
+[Dell PowerFlex](https://www.dell.com/en-us/dt/storage/powerflex.htm) is a software-defined storage solution from [Dell Technologies](https://dell.com). Among other things it offers the consumption of redundant block storage across the network.
+
+LXD offers access to PowerFlex storage clusters by making use of the NVMe/TCP transport protocol.
+In addition, PowerFlex offers copy-on-write snapshots, thin provisioning and other features.
+
+To use PowerFlex, make sure you have the required kernel modules installed on your host system.
+On Ubuntu these are `nvme_fabrics` and `nvme_tcp` which come bundled in the `linux-modules-extra-$(uname -r)` package.
+
+## Terminology
+
+PowerFlex groups various so called SDS (storage data servers) under logical groups within a protection domain.
+Those SDS are the hosts contributing storage capacity to the PowerFlex cluster.
+A protection domain contains storage pools which represent a set of physical storage devices from different SDS.
+In those storage pools LXD creates its volumes.
```

Comment:
```suggestion
PowerFlex groups various so-called {abbr}`SDS (storage data servers)` under logical groups within a protection domain.
Those SDS are the hosts that contribute storage capacity to the PowerFlex cluster.
A *protection domain* contains storage pools, which represent a set of physical storage devices from different SDS.
LXD creates its volumes in those storage pools.
```

We use italics to introduce new terminology. "SDS" should really be italics as well, but unfortunately we can't combine that with the `{abbr}`, which I think is nicer to have here. :)